linera-core/src/chain_worker/actor.rs

```diff
@@ -85,6 +85,7 @@ where
     StageBlockExecution {
         block: ProposedBlock,
         round: Option<u32>,
+        published_blobs: Vec<Blob>,
```

Comment:
why is this added now?

Reply:
Up until now we didn't need the published blobs inside `ChainStateView::execute_block`: We were checking their size before that, during execution only needed to make a note of their IDs (not their content), and then we store them after execution.

Now we handle fees (and the size check) inside `execute_block`.